README.md

```diff
@@ -124,100 +121,144 @@ The following properties are accepted:
 
-_Type_: `string`
+- `bundlerWarnings`: `Array<object>`
 
-Either `"js"`, `"go"`, or `"rs"`.
+  Contains any warnings that were generated by the bundler when preparing the function.
+
+- `config`: `object`
+
+  The user-defined configuration object that was applied to a particular function.
+
+- `inputs`: `Array<string>`
+
+  A list of file paths that were visited as part of the dependency traversal. For example, if `my-function.js` contains
+  `require('./my-supporting-file.js')`, the `inputs` array will contain both `my-function.js` and
+  `my-supporting-file.js`.
+
+- `nativeNodeModules`: `object`
+
+  A list of Node modules with native dependencies that were found during the module traversal. This is a two-level
+  object, mapping module names to an object that maps the module path to its version. For example:
+
+  ```json
+  {
+    "nativeNodeModules": {
+      "module-one": {
+        "/full/path/to/the/module": "1.0.0",
+        "/another/instance/of/this/module": "2.0.0"
+      }
+    }
+  }
+  ```
+
+- `nodeModulesWithDynamicImports`: `Array<string>`
+
+  A list of Node modules that reference other files with a dynamic expression (e.g. `require(someFunction())` as opposed
+  to `require('./some-file')`). This is an array containing the module names.
 
 ## zipFunction(srcPath, destFolder, options?)
 
-`srcPath`: `string`\
-`destFolder`: `string`\
-`options`: `object?`\
-_Return value_: `object | undefined`
+- `srcPath`: `string`
+- `destFolder`: `string`
+- `options`: `object?`
+- _Return value_: `object | undefined`
 
 ```js
 const { zipFunction } = require('@netlify/zip-it-and-ship-it')
 
 const zipNetlifyFunctions = async function () {
   const archive = await zipFunctions('functions/function.js', 'functions-dist')
+
   return archive
 }
 ```
 
 This is like [`zipFunctions()`](#zipfunctionssrcfolder-destfolder-options) except it bundles a single Function.
 
-The return value is `undefined` if the Function is invalid.
+The return value is `undefined` if the function is invalid.
 
 ## listFunctions(srcFolder)
 
-`srcFolder`: `string`\
-_Return value_: `Promise<object[]>`
+- `srcFolder`: `string`
+- _Return value_: `Promise<object[]>`
 
-Returns the list of Functions to bundle.
+Returns the list of functions to bundle.
 
 ```js
 const { listFunctions } = require('@netlify/zip-it-and-ship-it')
 
 const listNetlifyFunctions = async function () {
   const functions = await listFunctions('functions/function.js')
+
   return functions
 }
 ```
 
 ### Return value
 
-Each object has the following properties.
-
-#### name
+Each object has the following properties:
 
-_Type_: `string`
+- `name`: `string`
 
-Function's name. This is the one used in the Function URL. For example, if a Function is a `myFunc.js` regular file, the
-`name` is `myFunc` and the URL is `https://{hostname}/.netlify/functions/myFunc`.
+      Function's name. This is the one used in the Function URL. For example, if a Function is a `myFunc.js` regular file, the
```

Comment:
Not sure why, but this is rendered as:
![image](https://user-images.githubusercontent.com/26760571/118640520-bd83fb00-b7e1-11eb-90c7-0104acacd0e8.png)

Reply:
Uh oh! Fixed in https://github.com/netlify/zip-it-and-ship-it/pull/471/commits/2099d567fd550aafeb4d998ed057190352a40607. Thanks!